packages/instrumenter/src/disable-type-checking.ts

```diff
@@ -0,0 +1,47 @@
+import { types } from '@babel/core';
+import { File, Range } from '@stryker-mutator/api/core';
+import { notEmpty } from '@stryker-mutator/util';
+
+import { InstrumenterOptions } from './instrumenter-options';
+import { createParser } from './parsers';
+import { AstFormat } from './syntax';
+
+export async function disableTypeChecking(file: File, options: InstrumenterOptions) {
+  const parse = createParser(options);
+  const ast = await parse(file.textContent, file.name);
+  switch (ast.format) {
+    case AstFormat.JS:
+    case AstFormat.TS:
+      return new File(file.name, `// @ts-nocheck\n${removeTSDirectives(file.textContent, ast.root.comments)}`);
```

Comment:
nit: I find that a bit hard to read and am not so fond of the hardcoding here. How about moving the whole template literal part into a utility function?

Reply:
Done that 👍